marklogic-data-hub-central/ui-custom/src/views/Detail.tsx

```diff
@@ -180,128 +181,164 @@ const Detail: React.FC<Props> = (props) => {
+                        COMPONENTS[config?.detail?.entities[entityType]?.relationships.component],
+                        {config: config?.detail?.entities[entityType]?.relationships.config, data: detailContext.detail}, null
+                      )}
+                  </div>
+                </Section>
+              }
+
+              {config?.detail?.entities[entityType]?.imageGallery &&
+                <Section title="Image Gallery"
+                  collapsible={true}
+                  expand={expandIds.imageGallery}
+                  onExpand={() => {handleExpandIdsClick("imageGallery", true);}}
+                  onCollapse={() => {handleExpandIdsClick("imageGallery", false);}}
                 >
-                  {config.detail.entities[entityType]?.timeline.component && config.detail.entities[entityType]?.timeline.config &&
-                          React.createElement(
-                            COMPONENTS[config.detail.entities[entityType].timeline.component],
-                            {config: config.detail.entities[entityType].timeline.config, data: detailContext.detail}, null
-                          )}
+                  {config.detail.entities[entityType]?.imageGallery.component && config.detail.entities[entityType]?.imageGallery.config &&
+                    React.createElement(
+                      COMPONENTS[config?.detail?.entities[entityType]?.imageGallery.component],
+                      {config: config?.detail?.entities[entityType]?.imageGallery.config, data: detailContext.detail}, null
+                    )
+                  }
                 </Section>
-              </div>
-            </div>}
-          </div>
+              }
 
+            </div>
+          </div>
+          {config?.detail?.entities[entityType]?.timeline && <div className="row">
+            <div className="col-12">
+              <Section
+                title="Timeline"
+                data-test="timelineSection"
+                collapsible={true}
+                expand={expandIds.timeline}
+                onExpand={() => {handleExpandIdsClick("timeline", true);}}
+                onCollapse={() => {handleExpandIdsClick("timeline", false);}}
+              >
+                {config.detail.entities[entityType]?.timeline.component && config.detail.entities[entityType]?.timeline.config &&
+                  React.createElement(
+                    COMPONENTS[config.detail.entities[entityType].timeline.component],
+                    {config: config.detail.entities[entityType].timeline.config, data: detailContext.detail}, null
+                  )}
+              </Section>
+            </div>
+          </div>}
         </div>
 
-      ) : <Loading />}
+      </div>
+    );
+  };
+
+
+  return (
+
+    <div className="detail">
+
+      {config?.detail && !_.isEmpty(detailContext.detail) && !detailContext.loading ?
+        (
+          !config?.detail?.entities[entityType] ? getDetailNoEntity() : getDetailEntity
```

Comment:
With the example project loaded and flag set to true, the Detail page now breaks: 

https://watch.screencastify.com/v/GdnrbUTtzUZbh5Fe52nO

I think the problem is here, make this fix (add missing parens):
```
!config?.detail?.entities[entityType] ? getDetailNoEntity() : getDetailEntity()
```